src/codegen/llvm/codegen_llvm.cc

```diff
@@ -688,6 +688,12 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const Call* op) {
       indices.push_back(i);
     }
     return builder_->CreateShuffleVector(v0, v1, indices);
+  } else if (op->is_intrinsic("broadcast16")){
```

Comment:
I think we may want to avoid using the string literals directly from both backend and frontend sides because it might be error prone or user unfriendly as the number of them increases. Instead  we can probably create a "mapping" or "enum" to do this. But again, this is fine for now.